options/append.go

```diff
@@ -0,0 +1,25 @@
+package options
+
+import (
+	stream_revision "github.com/EventStore/EventStore-Client-Go/streamrevision"
+)
+
+type AppendToStreamOptions struct {
+	revision stream_revision.ExpectedRevision
```

Comment:
Is the encapsulation needed here? User can anyway create an invalid struct by calling `AppendToStreamOptions{}` so it's not protecting us from anything.

Reply:
You are right